ksql-rest-model/src/main/java/io/confluent/ksql/rest/entity/HeartbeatRequest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.entity;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import java.util.Objects;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonSubTypes({})
+public class HeartbeatRequest {
+
+  /*@JsonSerialize(using = HostInfoSerializer.class)
+  @JsonDeserialize(using = HostInfoDeserializer.class)*/
+  private final HostInfoEntity hostInfo;
+  private final long timestamp;
+
+  @JsonCreator
+  public HeartbeatRequest(@JsonProperty("hostInfo") final HostInfoEntity hostInfo,
+                          @JsonProperty("timestamp") final long timestamp) {
+    this.hostInfo = hostInfo;
+    this.timestamp = timestamp;
```

Comment:
should we include time zone or are we assuming that all of the ksql servers will be running on servers configured with UTC time? (or can we, whenever creating these, make sure that we convert current time to UTC)

Reply:
UTC is pretty common in server deployments.. We could keep it this way for now?